packages/charts/test/turbo-mode.test.js

```diff
@@ -0,0 +1,188 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-chart.js';
+
+describe('turbo-mode warning', () => {
+  let chart;
+
+  beforeEach(async () => {
+    sinon.stub(console, 'warn');
+  });
+
+  afterEach(() => {
+    console.warn.restore();
+  });
+
+  function addSeries(numDataItems) {
+    const data = Array.from(Array(numDataItems).keys());
+    const series = fixtureSync(`<vaadin-chart-series values="[${data.join(',')}]"></vaadin-chart-series>`);
+    chart.appendChild(series);
+  }
+
+  function addSeriesWithJS(numDataItems) {
+    const data = Array.from(Array(numDataItems).keys());
+    chart.updateConfiguration({
+      series: [
+        {
+          type: 'column',
+          data
+        }
+      ]
+    });
+  }
+
+  function addPoint(seriesIndex, data) {
+    chart.configuration.series[seriesIndex].addPoint(data);
+  }
```

Comment:
Would suggest moving `addPoint` into the `adding points` suite as there is no real need for keeping it in the global scope.

Reply:
Done